service/history/workflow/update/update.go

```diff
@@ -72,16 +88,23 @@ func (u *Update) accept() {
 	u.state = stateAccepted
 }
 
-func (u *Update) sendComplete(o *updatepb.Outcome) {
+func (u *Update) setOutcome(o *updatepb.Outcome) {
```

Comment:
I would call these two methods: `complete` and `rejected` similar to `accept` above. And `notifyResult` just `notify`.